.gitignore

```diff
@@ -1 +0,0 @@
-.idea
```

Comment:
Leave this unchanged, otherwise people using an IDE will get this folder added to the repository